src/Traits/RelativeKeywordTrait.php

```diff
@@ -62,4 +62,24 @@ public static function hasRelativeKeywords(?string $time): bool
 
         return false;
     }
+
+    /**
+     * Determines if there is no fixed date in the time string.
+     *
+     * @param \DateTimeInterface|string|null $time The time string to check
+     * @return bool true if doesn't contain a fixed date
+     */
+    private static function isRelativeOnly($time): bool
+    {
+        if ($time === null) {
+            return true;
+        }
+
+        if (!is_string($time)) {
+            return false;
+        }
+
+        // must not contain fixed date before relative keywords or time expression
+        return preg_match('/[0-9]{4}-[0-9]{1,2}-[0-9]{1,2}/', $time) !== 1;
```

Comment:
How does this pattern check for relative keywords?

Reply:
It does not.  This checks for an explicit date.  Anything other than an fixed date is assumed to be relative to TestNow.

Strings such as '2001-01-01 +10 days' will be sent to the constructor as they are not relative.